src/source/campaign-status.rst

```diff
@@ -0,0 +1,52 @@
+.. _campaign_status_overview:
+
+Campaign Status
+===============
+
+Each campaign can be in one of the following states. Some of the states can be set by the user, others
+are set by the system or by customer service.
+ 
+User Controlled States
+-------------------------
+
+.. _user-controlled-states:
+
+
+.. list-table::
+ :widths: 20 80
+ :header-rows: 1
+
+ * - Status
+   - Description
+
+ * - **ACTIVE**
+   - The campaign is active and its linked ads are shown on the site.
```

Comment:
it's active linked ads? or do we want to keep that out of this explanation?

Reply:
ooh good catch, no links mention :X